pyvrp/cpp/ProblemData.h

```diff
@@ -12,14 +13,14 @@ class ProblemData
 public:
     struct Client
     {
-        int x;                 // Coordinate X
-        int y;                 // Coordinate Y
-        int demand;            // Demand
-        int serviceDuration;   // Service duration
-        int twEarly;           // Earliest arrival (when using time windows)
-        int twLate;            // Latest arrival (when using time windows)
-        int prize = 0;         // Prize collected when visiting this client
-        bool required = true;  // Must this client be part of a solution?
+        int const x;                          // Coordinate X
+        int const y;                          // Coordinate Y
```

Comment:
I opted for `Coordinate`, mostly to avoid having things like `client.x + data.dist(0, 1)` compile, since I could not think of a case where such a thing is not an error. It's still possible now, but would require an explicit cast.